File: Primeira lista pt2/Exercicio15.java
public class Exercicio15 {
    public static void main(String[] args) {

        double total = 0;

        for (int dia = 1; dia <= 12; dia++) {

            double deposito = dia * 2;

            total += deposito;
        }

        System.out.println(
                "Total acumulado: R$ " + total
        );
    }
}

File: lista pt2/Exercicio15.js
let total = 0;
let deposito = 2;

for (let dia = 1; dia <= 12; dia++) {

    total = total + deposito;

    deposito = deposito + 2;
}

console.log(
    "Valor total acumulado: R$ " +
    total.toFixed(2)
);